Fix unpaired surrogate in test (#1072)
unicode-org/message-format-wg: test/README.md

# Unicode MessageFormat Test Suite

These test files are intended to be useful for testing multiple different _message_ processors in different ways:

- `syntax.json` — Test cases that do not depend on any registry definitions.

- `syntax-errors.json` — Strings that should produce a Syntax Error when parsed.

> [!NOTE]

> > `syntax-errors.json` does not include tests for the disallowed uses of unpaired surrogate code points

Reviewer: It's not relevant where these tests are not.
```suggestion
> Tests for the disallowed uses of unpaired surrogate code points are not included
```

> because JSON does not permit unpaired surrogate code points.
> If your implementation uses UTF-16 based strings (such as JavaScript `String` or Java `java.lang.String`)
> or otherwise allows unpaired surrogates in text or literals, you will need to implement tests equivalent
> to the following for syntax errors:
> ```json

> > { "src": "{\ud800}" }

Reviewer: This should be a fully qualified test, rather than relying on the `syntax-errors.json` `"defaultTestProperties"`.
```suggestion
> {
>   "locale": "en-US",
>   "src": "{\ud800}",
>   "expErrors": [{ "type": "syntax-error" }]
> }
```
